pkg/api/index.go

```diff
@@ -69,8 +69,18 @@ func (hs *HTTPServer) setIndexViewData(c *contextmodel.ReqContext) (*dtos.IndexV
 	// Locale is used for some number and date/time formatting, whereas language is used just for
 	// translating words in the interface
 	acceptLangHeader := c.Req.Header.Get("Accept-Language")
+	acceptLangHeaderFirstValue := ""
+
+	if len(acceptLangHeader) > 0 {
+		parts := strings.Split(acceptLangHeader, ",")
+		acceptLangHeaderFirstValue = parts[0]
+	}
+
 	locale := "en-US" // default to en formatting, but use the accept-lang header or user's preference
-	var regionalFormat string
+	if acceptLangHeaderFirstValue != "" {
+		locale = acceptLangHeaderFirstValue
+	}
```

Comment:
`locale` is tagged for removal on line 164, as `language` and `regionalFormat` are used for all the purposes it could potentially be used for. i'm not sure if it's even needed for backwards compat

Reply:
I structured the code this way to remove all the `locale` lines completely when we remove the feature toggle and it's the only behaviour. Does that make sense?

Reply:
Yep! Looks good then